pkg/backend/display/display.go

```diff
@@ -40,10 +40,10 @@ func printViewLive(out io.Writer, opts Options, permalink string) {
 	printPermalink(out, opts, "View Live", permalink)
 }
 
-// printOpenInBrowser print's an update's permalink prefaced with `Open in Browser (Ctrl+O): `. This is printed in
+// printOpenInBrowser print's an update's permalink prefaced with `View in Browser (Ctrl+O): `. This is printed in
 // interactive scenarios that use the tree renderer.
 func printOpenInBrowser(term terminal.Terminal, opts Options, permalink string) {
```

Comment:
might as well rename the function too:

```suggestion
func printViewInBrowser(term terminal.Terminal, opts Options, permalink string) {
```

Reply:
Done